src/main/java/com/maxi/calendar/service/UserServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.maxi.calendar.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.maxi.calendar.model.User;
+import com.maxi.calendar.repository.UserRepository;
+
+@Service
+public class UserServiceImpl implements IUserService {
+	
+	@Autowired
+	UserRepository userRepository;
```

Comment:
use constructor DI

Reply:
@NoArgsConstructor lombock annotation

Reply:
the previous fix was incorrect, then I change to:
UserServiceImpl(UserRepository userRepository){
		this.userRepository=userRepository;
	}